### ISSUE TITLE
Cannot deploy package with custom labels

### ISSUE THREAD

**Issue by mikesobczak** [forcedotcom/cli#695]
### Summary

I'm trying to deploy meta where there are custom labels, and it fails.

### Steps To Reproduce:

Create a custom label in org Setup
Reference the custom label in an Apex class (ex: Label.Sample_Label)
Include the custom label and the Apex class in manifest/package.xml
Retrieve the source for the custom label and the Apex class
Verify that the custom label and Apex class are in the project source folders
Attempt to push the meta to a different org (ex: sfdx force:source:deploy -x manifest/package.xml -u testsdo-20201006)
Get errors back stating that the custom labels could not be found.

Example output below:

PS C:\projects\VS Code\Test Meta Deploy> sfdx force:source:deploy -x manifest/package.xml -u testsdo-20201006
Job ID | 0AfB000000WhxjiKAB
SOURCE PROGRESS | ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ | 0/1 Components
TYPE   PROJECT PATH                                       PROBLEM
─────  ─────────────────────────────────────────────────  ───────────────────────────────────────────────────
Error  force-app\main\default\classes\TestClassLabel.cls  External string does not exist: Sample_Label (3:24)
ERROR running force:source:deploy:  Deploy failed.

### Expected result

Deploy succeeds

### Actual result

Deploy fails

### Additional information

N/A

**SFDX CLI Version**(to find the version of the CLI engine run sfdx --version): 

sfdx-cli/7.74.1-32db2396ed win32-x64 node-v12.18.3

**SFDX plugin Version**(to find the version of the CLI plugin run sfdx plugins --core)

@oclif/plugin-autocomplete 0.1.5 (core)
@oclif/plugin-commands 1.3.0 (core)
@oclif/plugin-help 3.2.0 (core)
@oclif/plugin-not-found 1.2.4 (core)
@oclif/plugin-plugins 1.9.0 (core)
@oclif/plugin-update 1.3.10 (core)
@oclif/plugin-warn-if-update-available 1.7.0 (core)
@oclif/plugin-which 1.0.3 (core)
@salesforce/sfdx-trust 3.4.3 (core)
alias 1.1.1 (core)
analytics 1.12.1 (core)
config 1.1.8 (core)
generator 1.1.3 (core)
salesforcedx 49.10.0 (core)
├─ templates 49.4.4 (core)
├─ @salesforce/sfdx-plugin-lwc-test 0.1.7 (core)
├─ custom-metadata 1.0.10 (core)
├─ salesforce-alm 49.11.0 (core)
└─ apex 0.0.9 (core)
sfdx-cli 7.74.1 (core)

**OS and version**:  Windows 10 Enterprise

**Comment by mikesobczak** [comment]
I completely uninstalled VS Code and the CLI and reinstalled both, and got the same error.

**Comment by mikesobczak** [comment]
Using an older version of the extension and the CLI on a different computer works fine.  I guess I can't update that installation because if I do, I'm dead in the water, unable to deploy code across orgs in my environment.

sfdx-cli/7.71.0-4193fb4acc win32-x64 node-v12.18.3

@oclif/plugin-autocomplete 0.1.5 (core)
@oclif/plugin-commands 1.3.0 (core)
@oclif/plugin-help 3.1.0 (core)
@oclif/plugin-not-found 1.2.4 (core)
@oclif/plugin-plugins 1.9.0 (core)
@oclif/plugin-update 1.3.10 (core)
@oclif/plugin-warn-if-update-available 1.7.0 (core)
@oclif/plugin-which 1.0.3 (core)
@salesforce/sfdx-trust 3.2.1 (core)
analytics 1.12.0 (core)
generator 1.1.3 (core)
salesforcedx 49.7.0 (core)
├─ salesforcedx-templates 49.4.0 (core)
├─ @salesforce/sfdx-plugin-lwc-test 0.1.7 (core)
├─ salesforce-alm 49.7.0 (core)
└─ custom-metadata 1.0.8 (core)
sfdx-cli 7.71.0 (core)

**Comment by mikesobczak** [comment]
Using the older installation worked just the one time.  VSCode was set to auto update, and stopped working when a new custom label was added to the actual package of code I'm working with.

**Comment by shetzel** [comment]
Duplicate of #652 and #673 

@mikesobczak - I added a partial fix for custom labels in the current release candidate.  Can you try installing that to see if it unblocks you?
`sfdx plugins:install salesforcedx@latest-rc`

It's a partial fix because it converts/deploys _all_ custom labels in the project versus just the labels defined in the manifest.  Converting/deploying specific labels is coming soon.

**Comment by mikesobczak** [comment]
That worked!

**Comment by clopezramos-everis** [comment]
I'm getting the same error on the same situation but on CI.
It's using version 7.99.

Only plugins installed sfdx-git-delta & sfdx-profiles-splitter.

@shetzel did the partial fix get pushed into latest release, do I have to install `salesforcedx@latest-rc` still?

**Comment by shetzel** [comment]
@clopezramos-everis - are you still on a specific version of the salesforcedx plugin?  I.e., if you run `sfdx plugins --core` does it have `(core)` after the salesforcedx plugin or a specific version?  If it's not the core version shipped with the CLI you should uninstall it to go back to the core version.  `sfdx plugins:uninstall salesforcedx`
